git-cliff-core/src/release.rs

```diff
@@ -177,6 +228,81 @@ impl Release<'_> {
 			None => Ok(config.get_initial_tag()),
 		}
 	}
+
+	/// Aggregates various statistics from the release data.
+	///
+	/// This method computes several metrics based on the current release and
+	/// its commits:
+	///
+	/// - Counts the total number of commits.
+	/// - Determines the number of days between the first and last commit.
+	/// - Counts the number of commits that follow the Conventional Commits
+	///   specification.
+	/// - Tallies how many times each link appears across all commit messages.
+	/// - Calculates the number of days since the previous release, if
+	///   available.
+	fn aggregate_statistics(&self) -> Statistics {
```

Comment:
Maybe we can change this function to `impl From<Release> for Statistics` (and move to `statistics` module) since it's simply constructing a `Statistics` from a `Release`. What do you think?